examples/flask/app/views.py

```diff
@@ -66,6 +66,9 @@ def get_current_user():
             graph = GraphAPI(result['access_token'])
             profile = graph.get_object('me')
 
+            if 'link' not in profile:
+                profile['link'] = "NotAvailable"
```

Comment:
I think it would be better to set this to either an empty string (`""`) or `None`.